server/utils/tasks/listTable.ts

```diff
@@ -28,6 +29,32 @@ const getTaskType = (task: Task): string => {
   return taskType
 }
 
+const getDecisionOutcome = (outcome: string): string => {
```

Comment:
I think I'd rather have `getDecisionOutcome` functions for both decision type, with either `AssessmentDecision`, `PlacementApplicationDecision` or `PlacementRequestTaskOutcome` passed in. That way you can have a record type like (for example):

```typescript
const assessmentDecisionTranslations: Record<AssessmentDecision, string> = {
    accepted: 'Accepted',
    rejected: 'Rejected'
}
```
 
And then you can have a getDecisionOutcome translator like so:

```typescript
const getDecisionOutcome = (outcome: AssessmentDecision): string => {
    return assessmentDecisionTranslations[outcome]
}
```

That way if any of the types change, we're alerted to this and can fix it.